### ISSUE TITLE
Export rom in .gbc when color mode only

### ISSUE THREAD

**Issue by attackemartin** [chrismaltby/gb-studio#1443]
**Is your feature request related to a problem? Please describe.**
Rom exports only do .gb files

**Describe the solution you'd like**
If the user selects "color mode only" in the settings, then the export could be .gbc

**Comment by chrismaltby** [comment]
Good idea. I've made this change now so this is how exporting will work in  v4.0.0-rc.2

**Comment by maxoakland** [comment]
That's a great idea

**Comment by attackemartin** [comment]
lovely! thanks!

**Comment by chunkysteveo** [comment]
I thought that was an odd thing not to be present - great if it's implemented now we have a "color only". 👍
